weekly-updates/Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -18,6 +19,89 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
+* BIDS Steering Group discussion about adding species to the standard was amazing.
+* All the lab mentorship notes on GitHub are bringing me so much joy 🙌
+* GA & JS gave a great talk at the Social Data Science special interest group!
+  And they were really well supported by lab members.
+* Really appreciated the conversion with the Exec Director ladies.
+* Super helpful mentorship call with KP 🌱
+* Really really no meeting Friday got through a lot of emails!
+  5513 --> 5392 (but then back up again by the time I got back from my dog walk 😂)
+  * https://twitter.com/kirstie_j/status/1367764683864875009?s=20
+* AB got some finances sorted out which I _hope_ will make one of our TPS project leads feel a bit better 🌼
+* I finished reading Thundehead and absolutely loved it.
+
+### What did you achieve
+
+* Presentation to the DECOVID Exec team.
+  Went well.
+  PHEW 😌
+* All hands meeting was helpful too.
+* Recruited 2 students to work on a project 🚀 🌟
+* Wrote letter of recommendation for PR.
+  Fingers crossed 🤞😬🤞
+* Got an ethics approval over the finish line.
+* Gave a report writing workshop for the DSG participants 📝
+* Research Fish submission.
+  * And the correction 🙈
+* I was helpful on a call with GT
+* I think I was helpful on a call with VF
+* Useful catch up chat with AM
+* Met the new fast streamer for TPS 👋
+* Brainstorming meeting for training at the Turing
+* Catch up with MBr
+* Bit of brainstorming on Trinity Challenge... but not really enough...
+* Bit of brainstorming for future resourcing for The Turing Way
+* 1:1 w AB, MS, EK, GA & ME
+* Reviewed the DECOVID newsletter!
+  Loved it!
+* Wrote huge complain about how the Turing slack is under utilised 😬
+* BIDS Steering Group meeting
+* EDoN meeting
+* I had 3 days off! 🙀
+  They were really great 🌟
+
+### What did you struggle with?
+
+* I should have helped someone more than I did.
+  It was a tough run because I also needed to fit my own oxygen mask.
+  But I'm sad at how things turned out.
+* EAG work was harder than it should have been.
+* I'm disappointed with engagement across the Turing in general.
+  I don't know what is up but there's something about 2021 that has just taken all the wind out of everyone's sails.
+  * I really wish we could find time to support each other _more_ not less as we continue to be so isolated 😢
+* The DSG workshop was super silent... I have absolutely no idea how it went 😩
+  * Apparently the September reports were some of the best they've ever had though so maybe I made a good impression??
+* Lab meeting discussion on arguments about barriers open science made me feel old & jaded.
+  I think that's ok though.... I am old and jaded from an open science perspective 😅
+* A few doc appts which took up some time and left me tired on one of the days.
+  Had a whole day of watching Deep Space 9.... which was fine I think.
+  It was what a needed 💤
+* Found out a grant that I was Co-I on wasn't funded.
+  Booooo
```

Comment:
Booooo indeed! I came across this thread and found it quite comforting https://twitter.com/maddi_pow/status/1292509832440885251